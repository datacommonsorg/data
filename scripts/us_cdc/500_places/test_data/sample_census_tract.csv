Year,StateAbbr,StateDesc,CountyName,CountyFIPS,LocationName,DataSource,Category,Measure,Data_Value_Unit,Data_Value_Type,Data_Value,Data_Value_Footnote_Symbol,Data_Value_Footnote,Low_Confidence_Limit,High_Confidence_Limit,TotalPopulation,Geolocation,LocationID,CategoryID,MeasureId,DataValueTypeID,Short_Question_Text
2018,AL,Alabama,Calhoun,01015,01015001600,BRFSS,Health Outcomes,All teeth lost among adults aged >=65 years,%,Crude prevalence,22.1,,,17.7,27.2,3791,POINT (-85.85970752 33.7049306),01015001600,HLTHOUT,TEETHLOST,CrdPrv,Teeth Loss
2018,AL,Alabama,Cherokee,01019,01019955900,BRFSS,Prevention,Mammography use among women aged 50-74 years,%,Crude prevalence,74.6,,,72.1,77.0,4235,POINT (-85.66863815 34.05057636),01019955900,PREVENT,MAMMOUSE,CrdPrv,Mammography
2018,AL,Alabama,Clay,01027,01027959000,BRFSS,Health Outcomes,Arthritis among adults aged >=18 years,%,Crude prevalence,40.9,,,40.0,41.7,3224,POINT (-85.87716485 33.30474877),01027959000,HLTHOUT,ARTHRITIS,CrdPrv,Arthritis
2018,AL,Alabama,Clay,01027,01027959200,BRFSS,Health Outcomes,Stroke among adults aged >=18 years,%,Crude prevalence,4.8,,,4.5,5.0,2480,POINT (-86.02087034 33.1899579),01027959200,HLTHOUT,STROKE,CrdPrv,Stroke
2018,AL,Alabama,Cleburne,01029,01029959600,BRFSS,Health Outcomes,Stroke among adults aged >=18 years,%,Crude prevalence,5.2,,,4.7,5.7,4202,POINT (-85.56228862 33.69258271),01029959600,HLTHOUT,STROKE,CrdPrv,Stroke
2018,AL,Alabama,Coffee,01031,01031010700,BRFSS,Unhealthy Behaviors,Obesity among adults aged >=18 years,%,Crude prevalence,36.0,,,35.3,36.7,2286,POINT (-86.028595 31.25325368),01031010700,UNHBEH,OBESITY,CrdPrv,Obesity
2018,AL,Alabama,Colbert,01033,01033020400,BRFSS,Unhealthy Behaviors,Obesity among adults aged >=18 years,%,Crude prevalence,32.1,,,30.4,34.1,1530,POINT (-87.72782325 34.74169033),01033020400,UNHBEH,OBESITY,CrdPrv,Obesity
2018,AL,Alabama,Conecuh,01035,01035960500,BRFSS,Health Outcomes,Stroke among adults aged >=18 years,%,Crude prevalence,6.4,,,6.0,6.9,1771,POINT (-87.00225928 31.46626901),01035960500,HLTHOUT,STROKE,CrdPrv,Stroke
2018,AL,Alabama,Covington,01039,01039962400,BRFSS,Prevention,Mammography use among women aged 50-74 years,%,Crude prevalence,76.7,,,73.4,79.7,3144,POINT (-86.23830634 31.35679532),01039962400,PREVENT,MAMMOUSE,CrdPrv,Mammography
