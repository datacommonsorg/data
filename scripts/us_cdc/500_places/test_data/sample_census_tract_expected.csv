Year,StateAbbr,StateDesc,CountyName,CountyFIPS,LocationName,DataSource,Category,Measure,Data_Value_Unit,Data_Value_Type,Data_Value,Data_Value_Footnote_Symbol,Data_Value_Footnote,Low_Confidence_Limit,High_Confidence_Limit,TotalPopulation,Geolocation,LocationID,CategoryID,MeasureId,DataValueTypeID,Short_Question_Text,Low_Confidence_Limit_StatVar,High_Confidence_Limit_StatVar,Population_StatVar,ConfidenceLevelTypeID
2018,AL,Alabama,Calhoun,1015,1015001600,BRFSS,Health Outcomes,All teeth lost among adults aged >=65 years,%,Crude prevalence,22.1,,,17.7,27.2,3791,POINT (-85.85970752 33.7049306),1015001600,HLTHOUT,dcs:Percent_Person_65OrMoreYears_WithAllTeethLoss,dcs:CrudePrevalence,Teeth Loss,dcs:LowerConfidenceIntervalLimit_Percent_Person_65OrMoreYears_WithAllTeethLoss,dcs:UpperConfidenceIntervalLimit_Percent_Person_65OrMoreYears_WithAllTeethLoss,dcs:SampleSize_Percent_Person_65OrMoreYears_WithAllTeethLoss,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Cherokee,1019,1019955900,BRFSS,Prevention,Mammography use among women aged 50-74 years,%,Crude prevalence,74.6,,,72.1,77.0,4235,POINT (-85.66863815 34.05057636),1019955900,PREVENT,dcs:Percent_Person_50To74Years_Female_ReceivedMammography,dcs:CrudePrevalence,Mammography,dcs:LowerConfidenceIntervalLimit_Percent_Person_50To74Years_Female_ReceivedMammography,dcs:UpperConfidenceIntervalLimit_Percent_Person_50To74Years_Female_ReceivedMammography,dcs:SampleSize_Percent_Person_50To74Years_Female_ReceivedMammography,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Clay,1027,1027959000,BRFSS,Health Outcomes,Arthritis among adults aged >=18 years,%,Crude prevalence,40.9,,,40.0,41.7,3224,POINT (-85.87716485 33.30474877),1027959000,HLTHOUT,dcs:Percent_Person_WithArthritis,dcs:CrudePrevalence,Arthritis,dcs:LowerConfidenceIntervalLimit_Percent_Person_WithArthritis,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithArthritis,dcs:SampleSize_Percent_Person_WithArthritis,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Clay,1027,1027959200,BRFSS,Health Outcomes,Stroke among adults aged >=18 years,%,Crude prevalence,4.8,,,4.5,5.0,2480,POINT (-86.02087034 33.1899579),1027959200,HLTHOUT,dcs:Percent_Person_WithStroke,dcs:CrudePrevalence,Stroke,dcs:LowerConfidenceIntervalLimit_Percent_Person_WithStroke,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithStroke,dcs:SampleSize_Percent_Person_WithStroke,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Cleburne,1029,1029959600,BRFSS,Health Outcomes,Stroke among adults aged >=18 years,%,Crude prevalence,5.2,,,4.7,5.7,4202,POINT (-85.56228862 33.69258271),1029959600,HLTHOUT,dcs:Percent_Person_WithStroke,dcs:CrudePrevalence,Stroke,dcs:LowerConfidenceIntervalLimit_Percent_Person_WithStroke,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithStroke,dcs:SampleSize_Percent_Person_WithStroke,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Coffee,1031,1031010700,BRFSS,Unhealthy Behaviors,Obesity among adults aged >=18 years,%,Crude prevalence,36.0,,,35.3,36.7,2286,POINT (-86.028595 31.25325368),1031010700,UNHBEH,dcs:Percent_Person_Obesity,dcs:CrudePrevalence,Obesity,dcs:LowerConfidenceIntervalLimit_Percent_Person_Obesity,dcs:UpperConfidenceIntervalLimit_Percent_Person_Obesity,dcs:SampleSize_Percent_Person_Obesity,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Colbert,1033,1033020400,BRFSS,Unhealthy Behaviors,Obesity among adults aged >=18 years,%,Crude prevalence,32.1,,,30.4,34.1,1530,POINT (-87.72782325 34.74169033),1033020400,UNHBEH,dcs:Percent_Person_Obesity,dcs:CrudePrevalence,Obesity,dcs:LowerConfidenceIntervalLimit_Percent_Person_Obesity,dcs:UpperConfidenceIntervalLimit_Percent_Person_Obesity,dcs:SampleSize_Percent_Person_Obesity,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Conecuh,1035,1035960500,BRFSS,Health Outcomes,Stroke among adults aged >=18 years,%,Crude prevalence,6.4,,,6.0,6.9,1771,POINT (-87.00225928 31.46626901),1035960500,HLTHOUT,dcs:Percent_Person_WithStroke,dcs:CrudePrevalence,Stroke,dcs:LowerConfidenceIntervalLimit_Percent_Person_WithStroke,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithStroke,dcs:SampleSize_Percent_Person_WithStroke,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Covington,1039,1039962400,BRFSS,Prevention,Mammography use among women aged 50-74 years,%,Crude prevalence,76.7,,,73.4,79.7,3144,POINT (-86.23830634 31.35679532),1039962400,PREVENT,dcs:Percent_Person_50To74Years_Female_ReceivedMammography,dcs:CrudePrevalence,Mammography,dcs:LowerConfidenceIntervalLimit_Percent_Person_50To74Years_Female_ReceivedMammography,dcs:UpperConfidenceIntervalLimit_Percent_Person_50To74Years_Female_ReceivedMammography,dcs:SampleSize_Percent_Person_50To74Years_Female_ReceivedMammography,dcs:CrudePrevalenceConfidenceLevel
